includes/apple-exporter/components/class-body.php

```diff
@@ -440,6 +439,42 @@ public function set_default_style() {
 		);
 	}
 
+	/**
+	 * Determine whether to apply a dropcap style for the component.
+	 * 
+	 * @param \Apple_Exporter\Theme $theme Object that stores theme level dropcap configuration.
+	 * @param string                $html The HTML to check for dropcap conditions. Should be the first paragraph of the post content.
+	 *
+	 * @return boolean
+	 */
+	private function dropcap_determination( $theme, $html ) {
+		// Toggle dropcap determination flag so that this logic applies only to the post's first paragraph.
+		$theme->dropcap_determined = true;
+
+		// Check that the theme is configured to apply dropcap styling.
+		if ( 'yes' !== $theme->get_value( 'initial_dropcap' ) ) {
+			return false;
+		}
+
+		// Check the first character for punctuation.
+		$content    = wp_strip_all_tags( $html );
+		$first_char = mb_substr( $content, 0, 1 );
```

Comment:
Initially tried `$content[0]`, but that won't work with multi-byte unicode characters.

Reply:
Good call.